src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -225,6 +235,21 @@ public Person getListingByName(Name name) {
                 .findFirst()
                 .orElse(null);
     }*/
+
+    /**
+     * Checks if there are any listings associated with the specified {@code seller}.
+     *
+     * @param seller The seller whose listings are to be checked.
+     * @return {@code true} if there is at least one listing associated with the seller;
+     *         {@code false} otherwise.
+     */
+    @Override
+    public boolean hasListingsForSeller(Person seller) {
+        requireNonNull(seller);
+        return listings.getListingList().stream()
+                .anyMatch(listing -> listing.getSeller().equals(seller));
```

Comment:
How does the .equals() check for Seller equality here? The exact same Seller object?